wiki/en/Unregistered-Servers.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: wiki
+title: "Running an Unregistered Server"
+lang: "en"
+permalink: "/wiki/Unregistered-Servers"
+---
+
+{% include breadcrumb.html root="More" branch1="Server Administration" branch1-url="Running-a-Server" %}
+
+# Running an Unregistered Server
+
+It is highly recommended to test your Server by registering it on one of the built-in Directories **first** so as to narrow down any subsequent problems in unregistered mode.
+
+## Setting up a Server behind a home router
+
+If you set up your server at home, you will probably need to change some settings in your router/firewall:
+
+### Port forwarding
+
+People from outside your home network will not be able to see things inside it. To let external Jamulus clients connect to your server, you need to set up port forwarding in your Router's settings. The exact setup differs for every router. For help see your Router's documentation or [portforward.com](https://portforward.com).
+
+**Note:** The default port for the current version of Jamulus is **UDP** (not TCP) port **22124**. You will usually forward the port **22124** from outside your network to the port **22124** of the machine running the Server.
```

Comment:
```suggestion
**Note:** The default port for Jamulus is **UDP** (not TCP) port **22124**. You will usually forward the port **22124** from outside your network to the port **22124** of the computer running the Server.
```

I believe we typically publish for the current version implicitly. Which version of Jamulus did not use 22124 as the port?